### ISSUE TITLE
Expanding only one row in react-bootstrap-table2

### ISSUE THREAD

**Issue by fastertabla** [react-bootstrap-table/react-bootstrap-table2#564]
In the react-bootstrap-table, i found a prop onlyOneExpanding which is used to allow expansion for only one row at any given point of time. I could not find a similar one react-bootstrap-table2. Could you please advice if there is a way?

Thanks
Ganesh

**Comment by AllenFang** [comment]
@fastertabla yes, we didn't support this feature so far, I will add it ASAP. thanks

**Comment by AllenFang** [comment]
@fastertabla 
https://react-bootstrap-table.github.io/react-bootstrap-table2/blog/2018/09/30/version-bump.html

Please upgrade to `react-bootstrap-table-next@1.2.0` at least.  [`expandRow.onlyOneExpanding`](https://react-bootstrap-table.github.io/react-bootstrap-table2/docs/row-expand-props.html#expandrowonlyoneexpanding-bool) will help you.  Please reopen this issue if the problem still remain, thank you

**Comment by fastertabla** [comment]
Thanks a lot Allen
